epub33/core/index.html

```diff
@@ -5494,6 +5515,32 @@ <h5>Inclusion</h5>
 								href="#sec-container-metainf-files"></a>.</p>
 					</section>
 
+					<section id="sec-parsing-urls-metainf">
+						<h5>Parsing URLs in the <code>META-INF</code> Directory</h5>
+
+						<p>To parse a URL string <var>url</var> used in files located in the <code>META-INF</code> directory the 
+							<a data-cite="url#concept-url-parser">URL Parser</a> MUST use the <a>container root URL</a> as <a data-cite="url#concept-base-url"><var>base</var></a> when applied to <var>url</var>.</p>
```

Comment:
Editorial nitpicking: I'd reword to the following, to be consistent with other calls of the URL parser:
```suggestion
							<a data-cite="url#concept-url-parser">URL Parser</a> MUST be applied to <var>url</var>, with the <a>container root URL</a> as <a data-cite="url#concept-base-url"><var>base</var>.</p>
```